microservices-orders/start.sh

```diff
@@ -10,17 +10,26 @@ check_netstat || exit 1
 check_running_elasticsearch 5.6.16 || exit 1
 check_running_kibana || exit 1
 check_running_cp ${CONFLUENT} || exit 1
+check_sqlite3 || exit 1
 
 ./stop.sh
 
 get_and_compile_kafka_streams_examples || exit 1
 
 confluent-hub install --no-prompt confluentinc/kafka-connect-jdbc:$CONFLUENT
 confluent-hub install --no-prompt confluentinc/kafka-connect-elasticsearch:$CONFLUENT
-echo "auto.offset.reset=earliest" >> $CONFLUENT_HOME/etc/ksqldb/ksql-server.properties
+grep -qxF 'auto.offset.reset=earliest' $CONFLUENT_HOME/etc/ksqldb/ksql-server.properties || echo 'auto.offset.reset=earliest' >> $CONFLUENT_HOME/etc/ksqldb/ksql-server.properties 
```

Comment:
This leaves one place for error: where the properties file has `auto.offset.reset=earliest` but is then overwritten later in the properties file with another value.

Reply:
I agree and thought of this as well, the previous code resulted in my properites file had many rows of the same entry after running the demo.  Any thoughts on handling these various cases?